templates/data/buildpack-data.yaml

```diff
@@ -203,7 +203,7 @@ templates:
                         - "postsubmit"
                   - jobConfig:
                       name: "pre-test-infra-prow-tools"
-                      run_if_changed: '^(go\.mod|go\.sum)$|^development/(tools|jobguard|image-url-helper)/((cmd|pkg)/.*\.go|.*\.toml|go\.mod)$|^prow/images/prow-tools/(Dockerfile|Makefile)$'
+                      run_if_changed: '^(go\.mod|go\.sum)$|^development/(tools|jobguard|image-url-helper)/((cmd|pkg)/.*\.go|.*\.toml|go\.mod)$|^development/markdown-index|^prow/images/prow-tools/(Dockerfile|Makefile)$'
```

Comment:
Please remove this, as you don't modify prow-tools image

Reply:
fixed